sonar-text-plugin/src/test/java/org/sonar/plugins/secrets/checks/CommonKeyCheckTest.java

```diff
@@ -42,7 +44,31 @@ public static void init() {
   @Test
   void shouldHaveRightAmountOfMatchers() {
     assertThat(check.ruleKey.rule()).isEqualTo("S6652");
-    assertThat(((SpecificationBasedCheck) check).getMatcher()).hasSize(38);
+    assertThat(((SpecificationBasedCheck) check).getMatcher()).hasSize(54);
   }
 
+  @Test
+  void shouldFindIssue() throws IOException {
+    String fileContent = "let requestHeaders: HTTPHeaders = [\n" +
+      "                \"Authorization\": \"Key d819f799b90bc8dbaffd83661782dbb7\",\n" +
+      "                \"Content-Type\": \"application/json\"\n" +
+      "            ]\n" +
+      "            \n" +
+      "            Alamofire.request(\n" +
+      "                \"https://api.clarifai.com/v2/models/c76825c96cbac79521a200cb54b7f09b/outputs\"";
+    assertThat(analyze(check, fileContent)).containsExactly(
+      "secrets:S6652 [2:38-2:70] Make sure this Clarifai key gets revoked, changed, and removed from the code.");
+  }
+
+  @Test
+  void shouldNotFindIssueBecausePatternAroundIsMissing() throws IOException {
+    String fileContent = "let requestHeaders: HTTPHeaders = [\n" +
+      "                \"Authorization\": \"Key d819f799b90bc8dbaffd83661782dbb7\",\n" +
+      "                \"Content-Type\": \"application/json\"\n" +
+      "            ]\n" +
+      "            \n" +
+      "            Alamofire.request(\n" +
+      "                \"https://api.x.com/v2/models/c76825c96cbac79521a200cb54b7f09b/outputs\"";
+    assertThat(analyze(check, fileContent)).isEmpty();
+  }
```

Comment:
I wanted test cases, so I would have differences when I remove the AuxiliaryPattern Matching. We can remove these after the Dynamic Test ticket, because it's just copied from a secret.